### DISCUSSION TITLE
XSLT conformance suite

### DISCUSSION THREAD

**Discussion by Devasta** [ballsteve/xrust#71] [Planning]
The XSLT test suite is enormous, but something we'll want to integrate eventually. Unlike the XML conformance suite, it is also being regularly updated.

Question: Do we add directly to the repo, or should we instead have a separate project which pulls in xRust and runs the tests from there?

**Comment by ballsteve** [comment]
Good question!

At this stage, my instinct is to say keep it separate. I can't see any benefit from including it in the crate; it will just make it enormously large and will not make testing any easier.

I reserve the right to change my mind later ;-)
